src/Plugins/LevelDBStore/Plugins/Storage/Store.cs

```diff
@@ -11,18 +11,24 @@
 
 using Neo.IO.Storage.LevelDB;
 using Neo.Persistence;
+using System.Collections;
 using System.Collections.Generic;
 
 namespace Neo.Plugins.Storage
 {
-    internal class Store : IStore
+    internal class Store : IStore, IEnumerable<KeyValuePair<byte[], byte[]>>
     {
         private readonly DB _db;
         private readonly Options _options;
 
         public Store(string path)
         {
-            _options = new Options { CreateIfMissing = true, FilterPolicy = Native.leveldb_filterpolicy_create_bloom(15) };
+            _options = new Options
+            {
+                CreateIfMissing = true,
+                FilterPolicy = Native.leveldb_filterpolicy_create_bloom(15),
+                CompressionLevel = CompressionType.SnappyCompression,
```

Comment:
Performance vs space?

Reply:
It's enabled already on default. I'm just ensure it. It will turn compression automatically if its can't compress.